pkg/analyzer/analyzer.go

```diff
@@ -42,6 +42,7 @@ var (
 	blueprintArtifactsRegexProperties = regexp.MustCompile("(\\s*\"properties\":)|(\\s*properties:)")
 	blueprintRegexTargetScope         = regexp.MustCompile("(\\s*\"targetScope\":)|(\\s*targetScope:)")
 	blueprintRegexProperties          = regexp.MustCompile("(\\s*\"properties\":)|(\\s*properties:)")
+	buildahRegex                      = regexp.MustCompile("buildah")
```

Comment:
Only `buildah` is too generic and may conflict with other files, I think we should use a different approach like:
```
 buildahRegex = regexp.MustCompile("\$\(\s*buildah\s+from\s+\w+\)")
```
that looks for the from command for buildah which every file is required to have